### PR TITLE
[Beginner Tutorial] Addresses #189 - Add pointer to file location info

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: tutorials/beginner/uis.md [hunk 1]
```diff
@@ -6,23 +6,24 @@ layout: tutorial-beginner
...
-OpenHAB 2 offers different UIs in its standard configuration: the _Paper UI_, the _Basic UI_ and the _Classic UI_ (this has to be installed manually from within Paper UI or via config file).
+OpenHAB 2 offers different UIs in its standard configuration: the *Paper UI*, the *Basic UI* and the *Classic UI* (this has to be installed manually from within Paper UI or via config file).
 
 ## The Paper UI
 
 The Paper UI is a new interface that helps setting up and configuring your openHAB instance.
 It does not (yet) cover all aspects, so you still need to resort to textual configuration files, but it already offers the following:
 
-- Add-on management: Easily install or uninstall openHAB add-ons
+-   Add-on management: Easily install or uninstall openHAB add-ons
```
Comment (Confectrician) [inline tutorials/beginner/uis.md]: Are those extra spaces needed? This comments addresses the following list items until line 23, too.

Comment (bgilmer77) [reply tutorials/beginner/uis.md]: Hi @Confectrician, according to the linter, you need two spaces after the dash mark.  I am assuming that @ThomDietrich has established this as the style he wants, so I just adjusted the text to get rid of the error.  Is that okay?

Comment (Confectrician) [reply tutorials/beginner/uis.md]: I still had luck yet with setting up my linter environment. 😄 
So if the rules say that, its ok for me and i would approve and merge the change then, if you are ready from your side.

Comment (bgilmer77) [reply tutorials/beginner/uis.md]: Yes please go ahead - the linter rules definitely say that the extra spaces should be there.  I checked this again last night when I saw your comment.

**Comment by Confectrician** [comment]
what about #606?
Should we wait for an answer or just merge this one already?

**Comment by bgilmer77** [comment]
It is a very small change - I can put it in now, but I was trying to be courteous.  Again, since I am still relatively new to the community, I don't want to step on anyone's work.  But if it was just me, I would take a moment, make a new commit and then note it in the other PR and close it.  WDYT?

**Comment by bgilmer77** [comment]
Error in last commit message - the last commit addresses #606.  I am going to merge this and close #606 with a pointer to this PR.

**Pull request merged**